Tests/DependencyInjection/AbstractMongoDBExtensionTest.php

```diff
@@ -294,7 +294,7 @@ public function testBundleDocumentAliases(): void
         $definition = $container->getDefinition('doctrine_mongodb.odm.default_configuration');
         $calls      = $definition->getMethodCalls();
         $this->assertTrue(isset($calls[0][1][0]['XmlBundle']));
-        $this->assertEquals('DoctrineMongoDBBundle\Tests\DependencyInjection\Fixtures\Bundles\XmlBundle\Document', $calls[0][1][0]['XmlBundle']);
+        $this->assertEquals('Doctrine\Bundle\MongoDBBundle\Tests\DependencyInjection\Fixtures\Bundles\XmlBundle\Document', $calls[0][1][0]['XmlBundle']);
```

Comment:
It looks like a good opportunity to start using `::class` everywhere :)

Reply:
This is a namespace name, not class :)

Reply:
Oops 👼